template/build/core/phing/tasks/deploy.xml

```diff
@@ -83,6 +83,8 @@
   <target name="deploy:commit">
     <!-- We make these commands quiet because they can cause the output to be so long that Travis CI stops logging. -->
     <exec command="git add -A" dir="${deploy.dir}" logoutput="true" passthru="true" checkreturn="true"/>
+    <!-- We are forcing vendor everywhere to add here in case a global git ignore flags it -->
+    <exec command="git add *vendor/* -f" dir="${deploy.dir}" logoutput="true" passthru="true" checkreturn="true"/>
```

Comment:
yeah I think adding only vendor should be fine. It will include subdirectories by default without the following asterisks, which could actually introduce the issue of ignoring hidden files.